### ISSUE TITLE
Confusion around Pipeline indication for snRNA-seq datasets

### ISSUE THREAD

**Issue by pecan88** [hubmapconsortium/portal-ui#1724]
Fill in as much as seems useful:

**Describe the issue**
Data provider confusion for submitted data with bracketed pipeline type in title and ideas on attribution between data providers and HIVE ...

**To Reproduce**
Look up Stanford TMC snRNA_seq datasets on Portal

**Expected behavior**
n/a

**Screenshots**

**Environment**
 - Browser: Chrome
 - Browser version: April 8, 2021 (current)
 - Portal version: `/static/VERSION` ... April 8, 2021

**Best guesses**
 - Relevant repos: not sure - likely affects pipelines and portal UI?
 - Developers to involve: Chuck, Matt, Bill; possibly Nils, Phil, Jonathan, Ziv

**Full emailed comment:**
After reviewing with D and S, we can confirm that our snRNAseq datasets that we uploaded are available and visible on the HuBMAP Data Portal. 

However, its a bit confusing to see snRNAseq, snRNAseq [Salmon] and snRNAseq (10X genomics) [Salmon]  all associated with Stanford TMC because Stanford TMC only uploaded the snRNAseq data type and HUBMAP DCC worked on the Salmon things. This lead to some confusion on my end because i had to wonder...did we mislabel our datasets or something? Did we click on a Salmon thing when labelling our datasets?

To remedy this I think it could be useful to change the “group” modifier possibly to say HUBMAP DCC next to the datasets that are analyzed by the DCC and not uploaded by the TMC…something like that.

Thank you,

A

**Comment by mccalluc** [comment]
I think the root concern is that it's hard for the TMC to find the data sets that they think of as "theirs" in among the processed datasets. Changing the group on derived datasets is one option, but there are others:
- Vis-lifting will hide some of the derived datasets over time.
- A new facet could be added to distinguish source and derived data.
- Creator could be made a sub-facet under Group, so it would look something like:
```
Stanford TMC       94
    Daniel Cotter  80
    HuBMAP Process 14
```

My inclination is to find a solution which doesn't require changing the Entity JSON.

I will put this on the agenda for the next UI meeting.

**Comment by khanshawPSC** [comment]
Ticket closed for this issue [#1217](https://on.spiceworks.com/tickets/open/1217/activity)

**Comment by mccalluc** [comment]
I can't find it in the notes, but my memory of this discussion was that, contrary to Standford's expectations, datasets that we derive from their submission still belong to them.

From the original report:
> However, its a bit confusing to see snRNAseq, snRNAseq [Salmon] and snRNAseq (10X genomics) [Salmon] all associated with Stanford TMC because Stanford TMC only uploaded the snRNAseq data type and HUBMAP DCC worked on the Salmon things.

I think that our response would be that the derived datasets are really just views on the TMC data, but the view is still theirs. If they are really interested in _responsibility_, that is captured by `Creator`.

@khanshawPSC / @pecan88 -- Closing / no action taken / but reopen if I'm mistaken, and up to you to communicate with Stanford, if that would be useful.
